### DISCUSSION TITLE
Different Lapis Dust amounts from macerating Lapis and Lapis Plate

### DISCUSSION THREAD

**Discussion by reoseah** [IndustrialCraftMC/IndustrialCraft2#62] [【漏洞反馈】Bugs Report]
If you place Lapis Lazuli into a Macerator, it takes 4 lapis for one dust. If you instead compress it to plates first, it only takes 1 lapis per dust. See images:
![2023-05-11_17 46 21](https://github.com/IndustrialCraftMC/IndustrialCraft2/assets/20323084/446b799f-8484-4f1c-963a-4a7aab38b31a)
![2023-05-11_17 46 10](https://github.com/IndustrialCraftMC/IndustrialCraft2/assets/20323084/55d2157a-9877-4ff4-8c70-8d8236dbbc71)

**Comment by AbbottMc** [comment]
more cost, less gain, it's life (x)
it seems caused by recipe, I will check it later
